libs/authjs-client/src/lib/gfw-client.ts

```diff
@@ -0,0 +1,365 @@
+const getUrl = (gateway: string, endpoint?: string) => `${gateway}${endpoint ? '/' + endpoint : ''}`
+
+const getAuthUrl = (gateway: string) => getUrl(gateway, 'auth')
+
+type GFWConfig = {
+  gatewayUrl: string
+  version?: string
+  client?: string
+  accessToken?: string
+  bearerToken?: string
+  refreshToken?: string
+}
+
+type Token = {
+  token: string
+  refreshToken: string
+}
+
+type RequestData = {
+  requestData: {
+    url: string
+    method: 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
+    body?: string
+  }
+  headers: Headers
+}
+const defaults: GFWConfig = {
+  gatewayUrl: 'https://gateway.api.globalfishingwatch.org',
```

Comment:
I'd add the env variable and use this as default in the same way we do [here](https://github.com/GlobalFishingWatch/frontend/blob/feature/VV-520-api-authentication-with-next/libs/api-client/src/api-client.ts#L12)